### PR TITLE
Webcrypto

### PR DESCRIPTION
**Pull request by daviddias**
~~**WIP**~~

Need to make sure js-multihashes is browser compatible too (because Buffer impl), so that then I can move this to use WebCrypto

### CODE REVIEW

File: tests/test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  t.end()
+})
+
+test('sha2-256', function (t) {
+  t.end()
+})
+
+test('sha2-512', function (t) {
+  t.end()
+})
```
Comment (masylum) [inline tests/test.js:24]: what's missing here?

Comment (daviddias) [inline tests/test.js:24]: finishing this one first - https://github.com/jbenet/js-multihash/pull/4

**Comment by daviddias** [comment]
@jbenet this ready to merge :) 

//cc @masylum

**Comment by daviddias** [comment]
thanks for `push` @jbenet  :)

**Pull request merged**